copybookreader-plugins/docs/CopyBookReader-batchsource.md

```diff
@@ -0,0 +1,99 @@
+# CopybookReader plugin
+
+
+Description
+-----------
+This is a source plugin that would allow users to read and process mainframe files defined using COBOL Copybook.
+This is the basic first implementation.
+
+Use Case
+--------
+
+It's basically used for reading flat file or dataset that is generated on a z/OS IBM mainframe based on a fixed length
+COBOL Copybook. This will also work on AS/400 computers.
+So, if a customer has flat files on HDFS that can be parsed using simple COBOL Copybook, then applying the Copybook
+one is able to read the file and it's fields easily.
```

Comment:
It's used for reading flat files or datasets generated on either a z/OS IBM mainframe or AS/400, 
based on a fixed-length COBOL Copybook. If you have a flat file on HDFS that can be parsed 
using a simple COBOL Copybook, then applying the copybook, one is able to easily read the file and its fields.